homework/generate-sequence/generateSequence.hpp

```diff
@@ -3,5 +3,13 @@
 
 std::vector<int> generateSequence(int count, int step) {
     // TODO: Implement me :)
-    return {};
+    std::vector<int> vec {step};
```

Comment:
Reserve memory for vector, when you know how many elements will be inserted. 
This will avoid the situation with resizing vector few times (which is always expensive).